server/services/messaging/message_service.py

```diff
@@ -181,9 +194,46 @@ def has_user_new_messages(user_id: int) -> dict:
         return dict(newMessages=new_messages, unread=count)
 
     @staticmethod
-    def get_all_messages(user_id: int):
+    def get_all_messages(user_id: int, locale: str, page: int, page_size=10, sort_by=None, sort_direction=None, message_type=None, from_username=None, project=None, task_id=None):
         """ Get all messages for user """
-        return Message.get_all_messages(user_id)
+        sort_by = sort_by or "date"
```

Comment:
Either here in the service function, or on the API call, we should check and sanitize the sort parameters